composer.json

```diff
@@ -40,7 +40,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "1.5.x-dev"
+            "dev-master": "1.6.x-dev"
```

Comment:
Yes, but currently it is not possible to use something like this in composer.json:
```json
"gos / web-socket-bundle": "1.6.*@dev"
```
If the next scheduled version is minor change then ```1.7``` is ok